packages/docusaurus/src/client/PendingNavigation.tsx

```diff
@@ -127,4 +127,4 @@ class PendingNavigation extends React.Component<Props, State> {
   }
 }
 
-export default withRouter(PendingNavigation);
+export default withRouter(PendingNavigation as any) as any;
```

Comment:
introducing `@types/react-router-dom` gave a fail here because the objects don't all match up - this is a known issue in the types for `withRouter` - it expects to be used as a decorator. I tried using a disable next error for this, but it still propagated into a bunch of places, so I just `as any`'d it

It previously was an any